src/components/ui/Hero/hero.scss

```diff
@@ -0,0 +1,93 @@
+$media: 1280px;
```

Comment:
I'm not sure how we're handling media queries but I remember seeing a merged PR related to it. Maybe there's more information in it on how to use them, with predefined Sass variables and stuff. Other people will probably know better.

Reply:
I also wonder if we're off by 1 pixel here. I'm getting the mobile version when my browser is at 1280px (which is half of my total screen width), where at this point it's usually considered fine for desktop.

![Screen Shot 2021-12-27 at 12 59 36](https://user-images.githubusercontent.com/381395/147488120-afbbb0fa-18ad-440e-877e-bc7990fb5e78.png)

Reply:
True, you can see this PR here: https://github.com/vtex-sites/base.store/pull/153